static/js/publisher/market/stickyListingBar.js

```diff
@@ -0,0 +1,20 @@
+export default function() {
+  const stickyBar = document.querySelector(".snapcraft-p-sticky");
```

Comment:
These could, in theory, match multiple elements. I just want to make sure that was intentional.